instrumentation/jaxrs/jaxrs-2.0/jaxrs-2.0-common/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/jaxrs/v2_0/JaxRsAnnotationsTracer.java

```diff
@@ -51,12 +52,16 @@ public Context startSpan(Context parentContext, Class<?> target, Method method)
     Span span = spanBuilder(parentContext, "jax-rs.request", INTERNAL).startSpan();
     updateSpanNames(
         parentContext, span, ServerSpan.fromContextOrNull(parentContext), target, method);
+    setCodeAttributes(span, target, method);
     return parentContext.with(span);
   }
 
   public void updateSpanNames(
       Context context, Span span, Span serverSpan, Class<?> target, Method method) {
-    String pathBasedSpanName = ServletContextPath.prepend(context, getPathSpanName(target, method));
+    String pathBasedSpanName = getPathSpanName(target, method);
+    if (pathBasedSpanName != null) {
```

Comment:
I'm not seeing how `pathBasedSpanName` can be null here?

Reply:
Thanks, that was careless of me.